daml/RealFac/CashAccount/CashAccount.daml

```diff
@@ -0,0 +1,118 @@
+module RealFac.CashAccount.CashAccount where
+
+template CashAccount
+  with
+    accountant : Party
+    owner : Party
+    balance : Int  -- total balance
+    reserved : Int  -- from total balance, reserved amount cannot be used
+  where
+    signatory owner, accountant
+    key (accountant, owner) : (Party, Party)
+    maintainer key._1
+
+    controller accountant can
```

Comment:
Yes! Thanks for the comment I was wondering about this myself, and deliberately wanted to explore classical banking cash transfer scenario a bit differently. As far as I can tell there is approach you mentioned which is also the same how I understood what is in the asset management tutorial layed out, and I was pursuing here another approach which is basically gives more privileges to the governor of the process (the accountant), and ensures consistency and soundness by the logic implemented int he template for creating the projects/contracts. I was going for the latter because it suits better this use case, and simpler from the contract model point of view (on the other side more complex on the contract implementation). But I wanted to explore the other path as well - due to lack of time this was sadly not possible.